content/java/concepts/calendar/terms/after/after.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.after'
+Description: 'Compare whether the current instance of Calendar occurs after the time represented by the specified Object.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+  - 'Time'
+  - 'Datetime'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+The **`.after`** method determine whether one date or time occurs chronologically after another date or time. It takes two instances of date or time objects as input and returns a boolean value; true if the first instance is later in time than the second instance, and false otherwise. This method is commonly used to compare and order events, appointments, or other time-related data.
+
+## Syntax
+
+```pseudo
+public boolean after(Object time);
```

Comment:
```suggestion
result = myCalendar.after(time);
```
We want pseudocode of the function in use in the syntax block, not the method signature. This should be followed by a sentence describing what the `result`, `myCalendar` and `time` are in use.